dominion-solutions/n8n
# Es/bjkeua/clockify trigger

# Details

Adding to ClockifyTrigger.nodes.ts to return information.

# Steps to test changes

1. Run "npm run dev" to view the website on your localhost.
1. Add Clockify trigger.
1. Add your Clockify API Key and set workspace to "Dominion Solutions".
1. Execute and make sure it returns the right table of information.

# Issues Addressed

[bjkeuna](https://app.clickup.com/t/bjkeua) - Pulling events from Clockify does not work anymore.

# Screenshots

![Screenshot at 2020-09-04 08-51-30](https://user-images.githubusercontent.com/60439807/92241242-e9499580-ee8b-11ea-8fb3-ca8f5e62ab2c.png)
